__tests__/commands/run.js

```diff
@@ -133,3 +133,16 @@ test('adds quotes if args have spaces and quotes', (): Promise<void> => {
     expect(execCommand).toBeCalledWith(...args);
   });
 });
+
+test('returns noScriptsAvailable and noBinAvailable with no bins and scripts', (): Promise<void> => {
+  return runRun([], {}, 'no-scripts', (config, reporter): ?Promise<void> => {
+    const rprtr = new reporters.BufferReporter({stdout: null, stdin: null});
+
+    // Emulate run output
+    rprtr.error(rprtr.lang('commandNotSpecified'));
+    rprtr.error(rprtr.lang('noBinAvailable'));
```

Comment:
Can you split this test so we test each case individually? This way we know we don't say `noScriptsAvailable` when we have them but we don't have any binaries for instance.